controllers/nemo_guardrails/nemoguardrail_controller.go

```
@@ -188,19 +188,26 @@ func (r *NemoGuardrailsReconciler) Reconcile(ctx context.Context, req ctrl.Reque
-		termination = utils.Reencrypt
-	}
-	routeConfig := utils.RouteConfig{
-		PortName:    nemoGuardrails.Name, // only one available port in the service, so don't need to specify any port name
-		ServiceName: nemoGuardrails.Name,
-		Termination: utils.StringPointer(termination),
-	}
-	err = utils.ReconcileRoute(ctx, r.Client, nemoGuardrails, routeConfig, routeTemplate, templateParser.ParseResource)
-	if err != nil {
-		utils.LogErrorReconciling(ctx, err, "route", nemoGuardrails.Name, nemoGuardrails.Namespace)
-		return ctrl.Result{}, err
+	if nemoGuardrails.Spec.ExposeRoute != nil && *nemoGuardrails.Spec.ExposeRoute == true {
```

golangci-lint (2.13.2)

[error] S1002: should omit comparison to bool constant, can be simplified to *nemoGuardrails.Spec.ExposeRoute

(staticcheck)